docs/modules/models/text_embedding/examples/elasticsearch.ipynb

```diff
@@ -14,6 +14,20 @@
     }
   },
   "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Testing ElasticsearchEmbeddings\n",
```

Comment:
this should be `##` (or it will show up as a Heading 1 which will mess up the toctree)

Reply:
we actually want first one to be Heading 1 (that's what page shows up as)